packages/dapp/src/components/FeatureCards/FeatureCard.tsx

```diff
@@ -109,6 +110,10 @@ export const FeatureCard = ({ data, isSuccess, assets }: FeatureCardProps) => {
     });
   };
 
+  const featureCardFallbackURL = appendUTMParametersToLink('https://li.fi/', {
+    utm_medium: 'feature_card',
```

Comment:
no campaign `jumper_to_lifi`

Reply:
It´s removed. That fallback was not gonna be used anyways since every feature card has the url set as required field.